packages/sit-onyx/src/styles/globals/layers.scss

```diff
@@ -0,0 +1 @@
+@layer onyx.reset, onyx.utility, onyx.component, onyx.density;
```

Comment:
What does this do / why is this needed? Maybe you can add a small comment

Reply:
done